### PR TITLE
enh(webpack-scaffold): improve prompt and doc

### PR DESCRIPTION
**Pull request by misterdev**
**What kind of change does this PR introduce?**
Slight improvement to the `webpack-cli init` user experience:
- improved defaults (see #788)
- improved messages

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Yes, also improving formatting and descriptions

**Summary**
#788

### CODE REVIEW

File: INIT.md [hunk 1]
```diff
@@ -3,74 +3,86 @@
...
-This answers to the [output directory](https://webpack.js.org/configuration/output/#output-path) of your application. The output directory is where servers or your `index.html` will read the generated bundle from.
+The output directory is where your bundled application will be. Your `index.html` will read the generated files from this folder, that is usually named `dist`.
 
 4. `Will you be using ES2015? (Y/n)`
 
-> *Property/key to resolve: [module.rules](https://webpack.js.org/configuration/module/#module-rules) (for .js files)*
+> *Property/key resolved: [module.rules](https://webpack.js.org/configuration/module/#module-rules) (for .js files)*
 
-If you answer `Yes` to this question, we will add [`ES2015`](https://babeljs.io/learn-es2015/) to your webpack configuration, which will allow you to use modern JavaScript in your project.
+This enables webpack to parse [`ES2015`](https://babeljs.io/learn-es2015/) code. Answer `Yes` if you to use modern JavaScript in your project.
```
Comment (ematipico) [inline INIT.md]: `if you want to use`

**Comment by misterdev** [comment]
Thanks @ematipico!

**Comment by misterdev** [comment]
I've found an incorrect example in `packages/webpack-scaffold/README.md` that wasn't covered by this PR. I've included it and taken the occasion to also update the `Confirm` API according to this PR and slightly improving formatting and consistency.

**Comment by ematipico** [comment]
Amazing contribution! Thank you!

**Pull request merged**